src/lib/mnemonics/generate-mnemonic.ts

```diff
@@ -0,0 +1,28 @@
+import { faker } from '@faker-js/faker';
```

Comment:
I'm not able to run the dev/index.ts file. I'm getting a ts error from this file for missing properties it looks like.
![Screenshot 2023-06-02 at 9 25 13 AM](https://github.com/RocketCommunicationsInc/astro-mock-data/assets/58487542/5fad1785-b0c7-4ab3-8c26-929627b9147e)

Reply:
It looks like I somehow pushed up the last changes without saving that file with the new props. I'll fix it.